bots/discord/OnMessageCreate.go

```diff
@@ -80,4 +81,62 @@ func OnMessageCreate(s *discordgo.Session, msg *discordgo.MessageCreate) {
 		s.ChannelMessageSend(msg.ChannelID, msg.Author.Mention()+" B-baaaaaaaka! Y..you...you want to...TOUCH MY CODE?!\n\nhttps://github.com/animenotifier/notify.moe/tree/go/bots/discord")
 		return
 	}
+
+	// Set the specific region role for the user
+	if strings.HasPrefix(msg.Content, "!region ") {
+
+		regions := map[string]string{
+			"africa":    "465876853236826112",
+			"america":   "465876808311635979",
+			"asia":      "465876834031108096",
+			"australia": "465876893036707840",
+			"europe":    "465876773029019659",
+		}
+
+		region := msg.Content[len("!region "):]
+
+		// check to make sure the region is in the region map
+		if _, ok := regions[region]; ok {
```

Comment:
Use early `return` instead of having huge if blocks.
https://github.com/animenotifier/notify.moe/blob/go/docs/code-style.md#early-returns